### PR TITLE
[MyTickets] functional components, hooks & css modules 

### LINKED ISSUE #2521: TicketsPage: functional components, css module & custom hooks
To try document better our incremental efforts on the two epics #2438 & #2439, I thought we could create sub-issue for each page we are planning to work on. 
After migrating some parts of the app, we could follow the same pattern:
 - class components -> functional components
 - `*.less` files -> css modules
 - redux connectors -> custom hooks 

For more insights checkout the parts which have been refactored: #2486, #2457

TicketsPage folder:
image

### LINKED ISSUE #2670: [My Tickets] Header looks weird with sidebar reduced
When the sidebar is reduced, the headers of the table looks weird: 

image

### PR DESCRIPTION
**Pull request by victorgcramos**
This diff closes #2670 and closes #2521 by refactoring MyTickets page to a functional approach, by using hooks and css modules.

### CODE REVIEW

**Review by vctt94 — CHANGES_REQUESTED** [review]
So, it looks like this PR breaks the overview tx history row:

image

**Comment by vctt94** [comment]
I also think it should show the tx name. If it is a vote, a ticket purchase, etc.

**Comment by victorgcramos** [comment]
@vctt94 I think we should use the home page overview mode on my tickets page when the width is collapsed... Check my changes and give me your thoughts! Thanks for the review!

**Review by vctt94 — CHANGES_REQUESTED** [review]
I agree and think that it looks good with the overview visual, but one thing, I noticed is the statistics page is broken now: 

image

File: app/components/views/TicketsPage/MyTicketsTab/MyTicketsTab.jsx [hunk 1]
```diff
@@ -1,120 +1,140 @@
...
 const labels = {
-  unknown: <T id="ticket.status.multiple.unknown" m="unknown" />,
-  unmined: <T id="ticket.status.multiple.unmined" m="unmined" />,
-  immature: <T id="ticket.status.multiple.immature" m="immature" />,
-  live: <T id="ticket.status.multiple.live" m="live" />,
-  voted: <T id="ticket.status.multiple.voted" m="voted" />,
-  missed: <T id="ticket.status.multiple.missed" m="missed" />,
-  expired: <T id="ticket.status.multiple.expired" m="expired" />,
-  revoked: <T id="ticket.status.multiple.revoked" m="revoked" />
+  [txTypes.UNKNOWN]: <T id="ticket.status.multiple.unknown" m="unknown" />,
```
Comment (vctt94) [inline app/components/views/TicketsPage/MyTicketsTab/MyTicketsTab.jsx:8]: Thanks for that!

**Review by vctt94 — COMMENTED** [review]
Thanks, it is mostly looking good. One thing I noticed is that the no more tickets bar has a different width: 

image

Beyond that I believe it is mostly looking good.

**Review by vctt94 — APPROVED** [review]
LGTM!

**Comment by amassarwi** [comment]
Referencing relevant parent issues:
#2438  #2439

**Comment by alexlyp** [comment]
needs a conflict resolved. then should be gtg

**Comment by amassarwi** [comment]
@victorgcramos Please reference the corresponding issue as well. 
Closes #2521

File: app/components/views/TicketsPage/MyTicketsTab/MyTicketsTab.jsx [hunk 2]
```diff
@@ -1,120 +1,140 @@
-import { ticketsList } from "connectors";
```
Comment (amassarwi) [inline app/components/views/TicketsPage/MyTicketsTab/MyTicketsTab.jsx:1]: Please delete the connector from connectors dir as well.

**Comment by amassarwi** [comment]
Also, there are few class components inside TicketsPage sub folder as well, they all need to be refactored to functional components as well.

File: app/components/views/TicketsPage/MyTicketsTab/MyTicketsTab.jsx (hunk 1 above)
Comment (amassarwi) [reply app/components/views/TicketsPage/MyTicketsTab/MyTicketsTab.jsx:8]: Indeed thanks 🔥 

Kill all these TODOs 🔪

Comment (amassarwi) [reply app/components/views/TicketsPage/MyTicketsTab/MyTicketsTab.jsx:8]: @victorgcramos

File: app/components/views/TicketsPage/MyTicketsTab/MyTicketsTab.jsx [hunk 3]
```diff
@@ -1,120 +1,140 @@
...
+    onChangeFilter({ listDirection: type.value });
+    setSortOrderKey(type.value);
+  };
+
+  const loadMoreThreshold = 90 + Math.max(0, window.innerHeight - 765);
+
+  return (
+    <TicketListPage
+      {...{
+        ...props,
```
Comment (amassarwi) [inline app/components/views/TicketsPage/MyTicketsTab/MyTicketsTab.jsx:120]: this isn't needed anymore, the props in this case was only the connector files, you can ditch it

**Review by alexlyp — APPROVED** [review]

**Pull request merged**